src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -3,6 +3,7 @@ class SendEventEmailsJob < ApplicationJob
 
   def perform(event_id)
     event = Event::Base.find(event_id)
```

Comment:
I would rather limit this to `Event::$WHATEVER CLASSES SEND MAILS`.find(event_id)

Reply:
I don't think there is a way to get `$WHATEVER CLASSES SEND MAILS`? Any class could send an email AFAIK.. Its not like the project log entry job where only certain classes create project log entries.